department-of-veterans-affairs/caseflow
# Queue | Add tooltip to hearing worksheet link

### Description
Adds a tooltip to the hearing worksheet link in case details view, explaining that the opened worksheet is non-editable. This also appends `?keep_open=true` to the link, which prevents the tab from closing when the user closes the print dialog

### Acceptance Criteria
- [ ] Hearing worksheet link has a tooltip

### Testing Plan
1. Go to `/hearings/dockets`, past docket day, open worksheet (`/hearings/##/worksheet`)
2. Go to `/queue`, case details view for that vet's case, confirm View Hearing Worksheet link appears within Hearings section
3. Confirm hearing worksheet link has a tooltip, confirm link destination has `?keep_open=true` at end of link

![screen shot 2018-09-14 at 2 41 08 pm](https://user-images.githubusercontent.com/8533004/45569006-44f85f80-b82c-11e8-966b-46ca9540433d.png)
